src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -822,15 +836,15 @@ public void InitializeSolutionBinding_Update_NotAllUpToDate_SomeProjectsUpdated(
             // Arrange
```

Comment:
I dont know if I should have left it with names, but as we test the output strings i figured it makes sense like this